UnityProject/Assets/Scripts/Shuttles/ShuttleConsole.cs

```diff
@@ -8,7 +8,7 @@
 /// <summary>
 /// Main component for shuttle console
 /// </summary>
-public class ShuttleConsole : NBHandApplyInteractable
+public class ShuttleConsole : MonoBehaviour, ICheckedInteractable<HandApply>
```

Comment:
Also a network behaviour here

Reply:
Same here, I think it should be fine. otherwise editor would give an error straight away

Reply:
Nope. You really only need nb if you are using syncvars or other nb features in a component. Quite a few components had it despite not needing it.